app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -42,4 +42,12 @@ def realm_field(f, can_override = false, override = false)
                 { :help_inline   => :indicator }
             ).html_safe
   end
+
+  def multiple_filter(hosts)
+    if multiple_with_filter?
+      no_filter   = _("Reminder: <strong> All #{hosts.size} hosts are selected </strong>").html_safe
+      with_filter = _("Reminder: <strong> All #{hosts.size} hosts are selected </strong> for query filter").html_safe + h(params[:search])
```

Comment:
In the helper file, I've couldn't use `total_entries` (no method exception),  instead I've used `size`.